docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -103,11 +103,30 @@ The preceding code demonstrates the following `System.ClientModel` protocol meth
 - Uses the `BinaryContent` type as a parameter to supply data for the request body.
 - Uses the `RequestContext` type to configure request options.
 - Returns data using the `ClientResult` type.
-- Calls the `GetRawResponse` method to access the response data.
+- Calls the <xref:System.ClientModel.ClientResult.GetRawResponse%2A> method to access the response data.
 
 > [!NOTE]
 > The preceding code configures the [ClientErrorBehaviors.NoThrow](/dotnet/api/system.clientmodel.primitives.clienterrorbehaviors) behavior for the `RequestOptions`. This option prevents non-success service responses status codes from throwing an exception, which means the app code should manually handle the response status code checks.
 
+A `System.ClientModel`-based response can be processed like a convenience model, if you take a dependency on `Azure.Core`. The following diff shows this alternative approach:
+
+```diff
+PipelineResponse response = result.GetRawResponse();
+
+- BinaryData output = result.GetRawResponse().Content;
+- using JsonDocument outputAsJson = JsonDocument.Parse(output);
+- JsonElement message = outputAsJson.RootElement
+-     .GetProperty("choices"u8)[0]
+-     .GetProperty("message"u8);
+
+- Console.WriteLine($@"[{message.GetProperty("role"u8)}]:
+-     {message.GetProperty("content"u8)}");
+
++ dynamic output = response.Content.ToDynamicFromJson();
++ var message = output.choices[0].message;
```

Comment:
nit: convenience models won't use lower-case property names, but if you pass propertyNameFormat: JsonPropertyNames.CamelCase as a parameter to ToDynamicFromJson, it will map JSON casing to .NET casing and let you use property names in the .NET style convention of lower case.  If you want to illustrate that here, it would support the statement that you can change less code when moving to convenience layer APIs later.